.env.example

```
@@ -0,0 +1,7 @@
+# 브라우저에 노출되는 변수는 반드시 NEXT_PUBLIC_ 접두사
+NEXT_PUBLIC_APP_URL=http://localhost:3000
+NEXT_PUBLIC_API_BASE_URL=https://api.example.com/api/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_BASE_URL key should go before the NEXT_PUBLIC_APP_URL key

(UnorderedKey)